### PR TITLE
Rename `@uifabric/example-data` to `@fluentui/example-data`

### PR DESCRIPTION
**Pull request by tomi-msft**
#### Pull request checklist

- [x] Addresses an existing issue: Part of #13384 
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Rename `@uifabric/example-data` to `@fluentui/example-data`

#### Focus areas to test

(optional)

### CODE REVIEW

File: change/@fluentui-react-icons-2020-10-20-10-54-27-set-version-rename.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Rename @uifabric/set-version to @fluentui/set-version",
+  "packageName": "@fluentui/react-icons",
```
Comment (ecraig12345) [inline change/@fluentui-react-icons-2020-10-20-10-54-27-set-version-rename.json:4]:
```suggestion
  "packageName": "@fluentui/react-icons-mdl2",
```

File: change/@uifabric-icons-2020-10-20-10-54-27-set-version-rename.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Rename @uifabric/set-version to @fluentui/set-version",
+  "packageName": "@uifabric/icons",
```
Comment (ecraig12345) [inline change/@uifabric-icons-2020-10-20-10-54-27-set-version-rename.json:4]:
```suggestion
  "packageName": "@fluentui/font-icons-mdl2",
```

**Comment by ecraig12345** [comment]
Looks like you need to manually update version.ts in font-icons-mdl2 and possibly also react-icons-mdl2.

**Comment by ecraig12345** [comment]
It's weird though that errors about version.ts would be showing up in this PR rather than the actual set-version one...?

**Comment by tomi-msft** [comment]
So what I think happened is I accidentally merged my master branch with set-version when I was trying to solve merge conflicts over there, and when I made this branch off of my master, it had the set-version rename already there. I think I will wait for build to pass for set-version PR, merge that, and then fix merge conflicts for this, to make things easier for myself

**Review by ecraig12345 — APPROVED** [review]

**Pull request merged**